examples/sphere/shallow_water.jl

```diff
@@ -256,6 +281,55 @@ elseif test_name == steady_state_compact_test_name
             local_geometry,
         )
 
+        return (h = h, u = u)
+    end
+elseif test_name == barotropic_instability_test_name
+    y0 = map(Fields.local_geometry_field(space)) do local_geometry
+        coord = local_geometry.coordinates
+
+        ϕ = coord.lat
+        λ = coord.long
+
+        if α == 0.0
+            ϕprime = ϕ
+        else
+            ϕprime = asind(sind(ϕ) * cosd(α) - cosd(ϕ) * cosd(λ) * sind(α))
+        end
+
+        # Set initial state of velocity field
+        uλprime(ϕprime) =
+            (u_max / eₙ) *
+            exp(1.0 / ((ϕprime - ϕ₀) * (ϕprime - ϕ₁))) *
```

Comment:
Also here, `ϕprime`, `ϕ₀`, `ϕ₁` are all radians in the paper.

Reply:
And I don't think scaling the parameters would work here. You may need to change the latitude variables. @simonbyrne?

Reply:
Oh, I think now I know what you mean. I think we need a conversion factor in the formula, like we did on this other [line](https://github.com/CliMA/ClimaCore.jl/blob/51a87e892b1eb49a4eea503de24e1fda81de2d3c/examples/sphere/shallow_water.jl#L312) for the h integral. So this line would read:

```
uλprime(ϕprime) =
            (pi / 180.0) * (u_max / eₙ) *
            exp(1.0 / ((ϕprime - ϕ₀) * (ϕprime - ϕ₁))) *
            (ϕ₀ < ϕprime < ϕ₁)
```
I'll try in a bit and will let you know

Reply:
It's not exactly the same. For the line you referred to, the term `quadgk(h_int, -90.0, ϕprime)[1]` when ϕprime is in radians is the same as  `(pi / 180.0) * quadgk(h_int, -90.0, ϕprime)[1]` when ϕprime is in degrees. But here, exp(1.0 / ((ϕprime - ϕ₀) * (ϕprime - ϕ₁))) when ϕ is in radians is **not** (pi / 180.0)*exp(1.0 / ((ϕprime - ϕ₀) * (ϕprime - ϕ₁))) when ϕ is in degrees, because ϕ is in the exponent. Same thing for eₙ.

Reply:
Oh, thank you for clarifying that. Now it is indeed more clear! I did not understand what you meant before. I'll look into it

Reply:
Sounds good, thanks! Let me know if there is anything still unclear.

Reply:
I think I finally fixed in [6245abb](https://github.com/CliMA/ClimaCore.jl/pull/292/commits/6245abb90a9a1e17c4e66a5380efb892304c91dd). Let me know if you have any other comments! And thanks again! 
This decision of using degrees over radians and between [-180.0, 180.0] over [0, 360.0] it's giving more than a hiccup in debugging test cases and examples when comparing them with the literature :joy:

Reply:
I think it's good now! Thanks!